src/includes/grouping/in-app/apple.mdx

```diff
@@ -1,6 +1,55 @@
+
 The sentry-cocoa SDK always marks public frameworks such as  UIKitCore, CoreFoundation, GraphicsServices, and so forth as `not inApp`.
-All private frameworks and the main executable inside the application bundle are marked as `inApp`. The following marks
-frames from libdispatch starting with `_dispatch_` as `inApp`.
+Before version 7.0.0, the sentry-cocoa SDK marks all private frameworks and the main executable inside the application bundle as `inApp`. Since version 7.0.0,
+the sentry-cocoa SDK sets the `inApp` flag only for frames originating from the main executable by using the [CFBundleExecutable].
+For private frameworks, such as Sentry, dynamic and static frameworks differ. If you are not familiar with these terms, we recommend
+reading [Dynamic Library Programming Topics].
+
+[CFBundleExecutable]: https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleexecutable
+[Dynamic Library Programming Topics]: https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/DynamicLibraries/000-Introduction/Introduction.html
+
+##### Dynamic Frameworks
+
+If you use dynamic frameworks, such as, for example, Sentry, the SDK marks these as `not inApp`. In case you have a private framework that should be `inApp`,
+you can use <PlatformLink to="/configuration/options/#in-app-include">inAppInclude</PlatformLink> or <PlatformLink to="/configuration/options/#in-app-exclude">inAppExclude</PlatformLink> of the SentryOptions.
+
+```swift {tabTitle:Swift}
+import Sentry
+
+SentrySDK.start { options in
```

Comment:
There might be another place to add these. Since InAppInclude/Exclude is a unified feature.
If we don't have it, we could eventually generalize this and add these snippets as includes. And fill out for Java/Android/Unity/.NET/etc

Reply:
I agree, I created an issue for that @bruno-garcia https://github.com/getsentry/sentry-docs/issues/3476.